### PR TITLE
feat (DDI-1259): Update popover position to be dynamic

### PR DESCRIPTION
**Pull request by amar-ABgov**

### CODE REVIEW

File: libs/docs/src/components/common/Popover.stories.mdx [hunk 1]
```diff
@@ -58,6 +58,40 @@ import {
   />
```
Comment (ArakTaiRoth) [inline libs/docs/src/components/common/Popover.stories.mdx:58]: There's no example for dynamic positioning in the Storybook?

File: libs/web-components/src/components/popover/Popover.svelte [hunk 2]
```diff
@@ -11,6 +12,15 @@
 
   let _isContentVisible = false;
   let _targetEl: HTMLElement;
+  let _popoverEl: HTMLElement;
+  let _popoverYPos: number = 0;
+
+  // margin
+  export let mt: Spacing = null;
```
Comment (ArakTaiRoth) [inline libs/web-components/src/components/popover/Popover.svelte:19]: These need to be added to the `*.html-data.json` file. Take a look at other files for an example.

**Review by ArakTaiRoth — APPROVED** [review]

**Review by vanessatran-ddi — APPROVED** [review]

**Pull request merged**